mozmeao-fr/bedrock-prod/ingress.yml

```diff
@@ -0,0 +1,56 @@
+apiVersion: cert-manager.io/v1alpha2
+kind: Issuer
+metadata:
+  name: letsencrypt
+  namespace: bedrock-prod
+spec:
+  acme:
+    # You must replace this email address with your own.
+    # Let's Encrypt will use this to contact you about expiring
+    # certificates, and issues related to your account.
+    email: it-se@mozilla.com
+    server: https://acme-v02.api.letsencrypt.org/directory
+    privateKeySecretRef:
+      # Secret resource used to store the account's private key.
+      name: letsencrypt
+    solvers:
+      - dns01:
+          cnameStrategy: Follow
+          route53:
+            region: us-west-2
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: bedrock-prod-ingress
+  namespace: bedrock-prod
+spec:
+  ports:
+  - port: 8000
+    targetPort: 8000
+  selector:
+    app: bedrock-prod
+    type: web
+---
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    cert-manager.io/issuer: letsencrypt
+  name: bedrock-prod
+  namespace: bedrock-prod
+spec:
+  rules:
+    - host: "bedrock.prod.fr.moz.works"
+      http:
+        paths:
+          - backend:
+              serviceName: bedrock-prod-ingress
+              servicePort: 8000
+            path: /
+  # This section is only required if TLS is to be enabled for the Ingress
+  tls:
+      - hosts:
+          - "bedrock.prod.fr.moz.works"
```

Comment:
does this also need to mention `www.mozilla.org`?

Reply:
I don't believe so.  Cloudflare terminates SSL and then makes requests to the origin dns address, bedrock-prod.moz.works. `bedrock-prod.frankfurt.moz.works` currently only has certs that are valid for moz.works.

Reply:
Ah, yeah, for TLS, right. We would need to configure the WMO domain for ingress routing though I'm guessing?

Reply:
@pmac I don't believe so I think cloudflare is directly accessing the origin, but it also doesn't costs me anything (no perf hits, not really that much extra work, etc) so I'm going to err on the side of caution and push a PR that includes it.  We can always pull that later if the logs don't show that host being used.